src/tpm12/tpm_crypto.c

```diff
@@ -48,6 +48,8 @@
 #include <openssl/rand.h>
 #include <openssl/sha.h>
 #include <openssl/engine.h>
+#include <openssl/evp.h>
+#include <openssl/rsa.h>
```

Comment:
Can you prefix your patch description with `tpm12:`, please. Otherwise LGTM.

Reply:
Done.